connect/service.go

```diff
@@ -63,3 +64,8 @@ func (svc service) checkRequeue(deviceUDID string) (int, error) {
 	}
 	return 0, nil
 }
+
+// Acknowledge Queries sent with DeviceInformation command
+func (svc service) ackQueryResponses() (error) {
```

Comment:
Not sure I understand what this is for.

Also, `(error)` should just be `error`. By convention you only need to surround it with `()` if there is more than one arg.